src/xdocs/google_style.xml

```diff
@@ -696,15 +696,30 @@
                 </td>
                 <td>
                   <span class="wrapper inline">
-                    <img src="images/ban_red.png" alt="" />
+                    <img src="images/ok_green.png" alt="" />
                   </span>
-                  This section's implementation is in progress, visit
-                  <a href="https://github.com/checkstyle/checkstyle/issues/15338">
-                    #15338
+                  <a href="checks/blocks/emptycatchblock.html#EmptyCatchBlock">EmptyCatchBlock</a>
+                  (<a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%20path%3A**%2Fgoogle_checks.xml+repo%3Acheckstyle%2Fcheckstyle+EmptyCatchBlock">
+                  config</a>)
+                  <br/>
+                  <br/>
+                  <span class="wrapper inline">
+                    <img src="images/ok_blue.png" alt="" />
+                  </span>
+                  <a href="checks/regexp/regexpsinglelinejava.html#RegexpSinglelineJava">RegexpSinglelineJava</a>
+                  (<a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%20path%3A**%2Fgoogle_checks.xml+repo%3Acheckstyle%2Fcheckstyle+RegexpSinglelineJava">
+                  config</a>)
+                  <br/>
+                  <br/>
+                  There are some false-negatives regarding K &amp; R style brackets
+                  for empty blocks, they're addressed at:
+                  <a href="https://github.com/checkstyle/checkstyle/issues/15338">#15338</a>
```

Comment:
I do not see false negatives, please remove.

Reply:
there are false-negatives:

https://github.com/checkstyle/checkstyle/blob/dbe82ad903d49e2a79592abe07e9789348dccb3c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java#L43-L482

https://github.com/checkstyle/checkstyle/blob/dbe82ad903d49e2a79592abe07e9789348dccb3c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java#L53-L61

we aren't able to detect K&R style blocks. I will fix this in my next PR

Reply:
Please keep false negative only on this line https://github.com/checkstyle/checkstyle/blob/dbe82ad903d49e2a79592abe07e9789348dccb3c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java#L44 , second line is not false negative, user has violation.

Reply:
https://github.com/checkstyle/checkstyle/blob/d5ac31dc7954c609703f04a959cc96c7c3f0d324/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java#L43-L48

removed `// false-negative` from violated code, kept those comments everywhere else though where there wasn't any violation. 

I intentionally kept those comments intentionally before because we want to give 2 violations:

- for having `{ }`
- for not having brackets in K & R style